composer.json

```diff
@@ -37,11 +37,11 @@
         "php-http/mock-client": "^1.6.0",
         "symfony/process": "^6.4|| ^7.0",
         "symfony/http-kernel": "^6.4|| ^7.0",
-        "phpunit/phpunit": "^9"
+        "phpunit/phpunit": "^10.5"
     },
     "conflict": {
         "toflar/psr6-symfony-http-cache-store": "<2.2.1",
-        "phpunit/phpunit": "<8"
+        "phpunit/phpunit": "<10"
```

Comment:
It is strange that I can not use the `3.x` package when my PHPUnit version is `9.6` which is still a widely. used version of PHPUnit.

I would like to avoid this conflict think most of the people will not require a test besed on the `WebServerSubscriber` and so this should only be a softrequirement.

Reply:
i think you are right. 

fixed in 24a47d586b81a7a9a21fe8aad88b20948e6856cf